src/routes/api/[user].ts

```diff
@@ -8,7 +8,22 @@ export async function get({ params }) {
       await fetch(`https://api.scratch.mit.edu/users/${user}/`)
     ).json();
     let scratchdb,
-      scratchdb_forum_user = { counts: undefined };
+      scratchdb_forum_user = {
+        counts: {
+          "total":{"count":0,"rank":1},
+          "Advanced Topics":{"count":0,"rank":1},
+        },
+        "history":[
+          {"date":"1234-12-23T12:34:56.000Z","value":1},
+        ],
+        "firstSeen":{
+          "date":"1234-12-23T12:34:56.000Z","id":1
+        },
+        "lastSeen":{
+          "date":"1234-12-23T12:34:56.000Z","id":1
+        },
+        "signature":"Well, that one fell off the database. ScratchDB didn't know what to say.",
+      };
```

Comment:
Copied the data from my profile.

Reply:
Hey, you should turn it into a Typescript interface!

Reply:
> Hey, you should turn it into a Typescript interface!

How?

Reply:
Example:

``ts
interface Person {
// name is type string
  name: string;
// age is type number
  age: number;
// next is optional
// type[] means it in array
  friends?: Person[];
}
```ts

Reply:
> Example:
> 
> ```ts
> interface Person {
> // name is type string
>   name: string;
> // age is type number
>   age: number;
> // next is optional
> // type[] means it in array
>   friends?: Person[];
> }
> ```

Okay???

Reply:
Nevermind